src/FloatingButton/FloatingButton.tsx

```diff
@@ -127,6 +127,14 @@ export default class FloatingButton extends React.PureComponent {
     }
   };
 
+  additionalButtonHandler(button) {
+    return () => {
+      const { active } = this.state;
+      this.setState({ active: !active });
```

Comment:
I would explicitly set to ```false``` instead of toggling, since that's a more accurate reflection of what the additional button handler will do

Reply:
makes sense